backend/app/Actions/Sessions/GetAvgSessionRequest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Actions\Sessions;
+
+use App\Http\Requests\Api\GetAvgSessionHttpRequest;
+
+final class GetAvgSessionRequest
```

Comment:
inherit from ButtonDataRequest. There is already exists method period, that covers all cases for start and end dates. This is also good, but just to be consistent with the whole project